### DISCUSSION TITLE
I disabled my account ( I want it back)

### DISCUSSION THREAD

**Discussion by YoussefGeorge-Dev** [saber-notes/saber#1185] [General]
I was sent an email that said 
"You can cancel your account deletion by contacting an administrator before the March 27, 2024."
I also have no idea of knowing my encryption password, does anyone know how??
(ps sorry if this is redundant)

**Comment by adil192** [comment]
I've gone ahead and re-enabled your account.
As for the encryption password, you choose it when you first log in to the Saber app

**Reply by YoussefGeorge-Dev** [reply]
thank you so much! i don't recall setting an encryption key tbh. is there anyway i can reset it?

**Reply by adil192** [reply]
![Screenshot_20240320-145026.png](https://github.com/saber-notes/saber/assets/21128619/3654c43d-0832-4c3b-9ca5-6623e606d86c)

**Reply by YoussefGeorge-Dev** [reply]
i cant seem to find a Saber folder on nextcloud, is that normal

**Reply by QubaB** [reply]
If you did not used synchronization to nextcloud before, then Saber folder is not on nextcloud.

It seems that you supposed you are syncing, but you did not.

**Reply by YoussefGeorge-Dev** [reply]
hmm how do i start with syncing then? since when i want to log on it asks for an encryption password

**Reply by QubaB** [reply]
You should create some password and you should remember it.  This password is used to cipher your files on server. The password is your choice.

**Reply by YoussefGeorge-Dev** [reply]
![image](https://github.com/saber-notes/saber/assets/134179178/899b3825-3ee1-4c61-8b5d-2ce6f4916b61)
i tried setting a password and this is what it says

**Reply by QubaB** [reply]
![Screenshot_4](https://github.com/saber-notes/saber/assets/74395874/6693d3a7-403b-4641-93ea-20437944f259)

Fill correctly your nextcloud server URL
put your username 
put your password to login to nextcloud server
put your encryption password (your choice).

Before you try, do it in your browser to see if you know correct username nad password.

When you press Login, Saber usualy shows that you are not logged, but if you exit settings page and return back, information is updated.

**Reply by YoussefGeorge-Dev** [reply]
thank you so much
